CHANGELOG.md

```diff
@@ -6,9 +6,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.0] - 2021-02-18
```

Comment:
* Should have caught it before, but we try to update the CHANGELOG in the PR introducing the change. That way someone doesn't have to go back and track everything down when preparing for the release.
* If you are going to include links, can they be markdown formatted instead of displaying the entire URL.
* It isn't bad, but we generally only document changes to the library that are user facing. This would exclude project management or docs changes.

Reply:
All of those make sense - I'll make those changes. Thanks for the feedback!